docs/tutorials/build-a-dapp/pallet.md

```diff
@@ -55,12 +55,18 @@ substrate-node-template
 +-- ...
 ```
 
-You will see some pre-written code that acts as a template for a new pallet. You can read over this
-file if you'd like, and then delete the contents since we will start from scratch for full
-transparency. When writing your own pallets in the future, you will likely find the scaffolding in
-this template pallet useful.
+You will see some pre-written code that acts as a template for a new pallet.
+When writing your own pallets in the future, you will likely find the scaffolding
+in this template pallet useful. But for the purposes of really learning how a pallet
+is constructed, **delete all contents of this file**.
 
-## Build Your New Pallet
+> The following sections will start _from scratch_ - assuming that you have cleared the
```

Comment:
This phrase seems repetitive to me: "We will start from scratch for full showing how to build a pallet from ground zero." I like the note here though, maybe try squash the two phrases together?

Reply:
open to your language 😁